packages/node/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -49,6 +53,15 @@ export type SentryHttpInstrumentationOptions = InstrumentationConfig & {
    */
   extractIncomingTraceFromHeader?: boolean;
 
+  /**
+   * Whether to propagate Sentry trace headers in ougoing requests.
+   * By default this is done by the HttpInstrumentation, but if that is not added (e.g. because tracing is disabled, ...)
```

Comment:
```suggestion
   * By default this is done by the HttpInstrumentation, but if that is not added (e.g. because tracing is disabled)
```

I think the extra ellipses can be confusing to users.